www/scripts/productlist/ProductSettingsView.js

```diff
@@ -267,6 +280,7 @@ function (declare, domAttr, domClass, domConstruct, Dialog, Button,
 
       var product = new CC_PROD_OBJECTS.ProductConfiguration({
         endpoint: args['endpoint'],
+        runLimit: args['runlimit'] ? args['runlimit'] : -1,
```

Comment:
How does `-1` harmonise with the logic behind this field? Can't we set this to `null` and send a `None` into Python?